Fatal error attempting to build FDO

My system was a NUC with a fresh install of Fedora 36 desktop. I checked out tag v0.4.9 and followed the build instructions located at: [https://github.com/fedora-iot/fido-device-onboard-rs/blob/v0.4.9/CONTRIBUTING.md](https://github.com/fedora-iot/fido-device-onboard-rs/blob/v0.4.9/CONTRIBUTING.md)

Per the instructions, I installed the listed pre-reqs and then executed the following build command:
`>$ cargo build --release`

After  a while, I received the following build error:

```
   ...
   Compiling chrono-tz v0.6.1
   Compiling is-terminal v0.4.5
   Compiling thread_local v1.1.4
error: failed to run custom build command for `devicemapper-sys v0.1.5`

Caused by:
  process didn't exit successfully: `/home/m*****/projects/fido-device-onboard-rs/target/release/build/devicemapper-sys-3043b94ff55de525/build-script-build` (exit status: 101)
  --- stderr
  dm.h:2:10: fatal error: 'libdevmapper.h' file not found
  thread 'main' panicked at 'Could not generate dm.h bindings: ClangDiagnostic("dm.h:2:10: fatal error: 'libdevmapper.h' file not found\n")', /home/mmillsap/.cargo/registry/src/github.com-1ecc6299db9ec823/devicemapper-sys-0.1.5/build.rs:24:10
  note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
warning: build failed, waiting for other jobs to finish...
```

After some searching on 'libdevmapper.h', a recommended package in which to find that header file was 'device-mapper-devel'. I installed that package on my system and the build was able to successfully complete.

--- a/CONTRIBUTING.md
+++ b/CONTRIBUTING.md
@@ -11,7 +11,11 @@ If you want to suggest any changes to the process, please feel free to open a Pu
 
 There are a number of ways in which you can set up development environments for working on FDO (FIDO Device Onboarding). 
 
-##### On Fedora host system
+#### On Fedora host system
+
+Please note that the build requirement are valid for the **latest stable Fedora release**.
+Previous Fedora releases *might* need a different set of dependencies than the ones outlined here.
+
 In order to make a test build of this crate, when using Fedora, you can run:
 
 ``` bash
@@ -24,11 +28,14 @@ cargo build --release
 After making changes, you can use `cargo test` to run the test suite, `cargo fmt` to ensure the code style is adhered to, and `cargo clippy` to check for some common lints against the code.
 
 
-##### On non-Fedora host system
+#### On non-Fedora host system
+
 If your host operating system is other than Fedora, following two ways are suggested for devlopement setup.
 1.VSCODE remote containers
 2.Toolbox 
-###### 1. VSCODE remote container setup:
+
+##### 1. VSCODE remote container setup:
+
 Pre-requisites:
 - Install VSCODE for your host operating system. 
 - Install ‘Remote-containers’ extension OR you can also install the ‘Remote Development’ extension which contains three extensions useful for remote container development.
@@ -46,13 +53,14 @@ cargo build
 cargo test 
 ``` 
 
-###### 2. ToolBox 
+##### 2. ToolBox
+
 ToolBox is an utility for containerized command line environments on Linux. So if you are working on RHEL and instead of creating another vm for fedora ‘ToolBox’ is a good option too. So by creating a container for fedora and then fdo can be built on it. Follow below commands to get build env up & running.
 
 ``` bash
 sudo yum install toolbox
-toolbox create --distro fedora --release f36
-toolbox enter fedora-toolbox-36 
+toolbox create --distro fedora
+toolbox enter fedora-toolbox-{VERSION}
 ```
 
 You can find the name of the container using : 'toolbox list' command.

